server/controllers/api/users/index.ts

```diff
@@ -197,11 +197,25 @@ async function createUser (req: express.Request, res: express.Response) {
     adminFlags: body.adminFlags || UserAdminFlag.NONE
   }) as MUser
 
+  // NB: due to the validator usersAddValidator, password==='' can only be true if we can send the mail.
+  const createPassword = userToCreate.password === ''
```

Comment:
`if (!userToCreate.password) userToCreate.password = await generateRandomString(20)` should be enough, and I don't think the comment is necessary

Reply:
I need the createPassword vars on line 211, to send the email